samplers/parser.go

```diff
@@ -122,29 +123,57 @@ func ConvertMetrics(m *ssf.SSFSpan) ([]UDPMetric, error) {
 // span and returns metrics that can be determined from that
```

Comment:
We should update the docstring here to clarify that it returns both indicator and objective metrics.